yaml/converter/legacy/internal/config.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/cncd/pipeline/pipeline/frontend/yaml/matrix"
```

Comment:
can we copy this package local to `github.com/drone/drone-yaml/yaml/converter/legacy/matrix`? I may eventually delete the cncd organization.